backend/inspirehep/snow/api.py

```diff
@@ -627,6 +612,17 @@ def get_formatted_user_list(self):
         ]
         return users
 
+    def _get_user_by_email(self, user_email):
+        found_user = next(
+            (
+                user["id"]
+                for user in self.get_formatted_user_list()
+                if user["email"] == user_email
+            ),
+            None,
+        )
+        return found_user
```

Comment:
should we add a `try/except`?

Reply:
would that make sense?

Reply:
Do you mean `StopIteration` instead of `None`? I prefer this way tbh :p